naming/src/main/java/com/alibaba/nacos/naming/core/ServiceManager.java

```diff
@@ -923,7 +923,7 @@ public int getInstanceCount() {
         }
         return total;
     }
-    
+
```

Comment:
Please do not change the indentation. Maybe you do not use nacos code style. Can you read the style/codeStyle.md in source code directory?